File: rafael/Atividade02.c
#include <stdio.h>
#include <locale.h>


int fatorial(int n) {
    int resultado = 1;
    for (int i = 1; i <= n; i++) {
        resultado *= i;
    }
    return resultado;
}

int main() {
	setlocale(LC_ALL, "Portuguese");
    int opcao, num;

    while (1) {
        printf("\n--- MENU ---\n");
        printf("1 - Calcular o quadrado de um número\n");
        printf("2 - Calcular o fatorial de um número\n");
        printf("3 - Sair\n");
        printf("Escolha uma opção: ");
        scanf("%d", &opcao);

        switch (opcao) {
            case 1:
                printf("Digite um número: ");
                scanf("%d", &num);
                printf("O quadrado de %d é %d\n", num, num * num);
                break;
            case 2:
                printf("Digite um número: ");
                scanf("%d", &num);
                printf("O fatorial de %d é %d\n", num, fatorial(num));
                break;
            case 3:
                printf("Saindo do programa...\n");
                return 0;
            default:
                printf("Opção inválida! Tente novamente.\n");
        }
    }

    return 0;
}

File: rafael/Atividade02.py
def fatorial(n):
    resultado = 1
    for i in range(1, n + 1):
        resultado *= i
    return resultado


while True:
    print("\n--- MENU ---")
    print("1 - Calcular o quadrado de um número")
    print("2 - Calcular o fatorial de um número")
    print("3 - Sair")

    opcao = int(input("Escolha uma opção: "))

    if opcao == 1:
        num = int(input("Digite um número: "))
        print(f"O quadrado de {num} é {num ** 2}")
    elif opcao == 2:
        num = int(input("Digite um número: "))
        print(f"O fatorial de {num} é {fatorial(num)}")
    elif opcao == 3:
        print("Saindo do programa...")
        break
    else:
        print("Opção inválida! Tente novamente.")

#Comparacao
# Em Python, o menu é mais simples e compacto, pois não precisamos do switch/case.
# Em C, o switch deixa claro os casos, mas exige mais código (declaração de funções, break etc.).
# Conclusão: Python é mais legível e rápido de implementar.